v0/destinations/ga/data/GAProductListEventConfig.json

```diff
@@ -1,6 +1,5 @@
 {
   "properties.list_id": "il1nm",
   "request_ip": "uip",
-  "context.userAgent": "ua",
```

Comment:
you could keep this in mapping files as well. When we will refactor the transformer code, it would be easier.